src/media.js

```diff
@@ -155,7 +134,7 @@
       },
 
       workerAudioSendOnmessage() {
-        workerAudioSend.onmessage = function (e) {
+        workerAudioSend.onmessage = (e) => {
```

Comment:
Codacy found an issue: ['workerAudioSend' is not defined.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=6361181)